src/components/AboutComponent/AboutComponent.tsx

```diff
@@ -28,20 +27,14 @@ const About: React.FC = () => {
                     <div className="about-layout__foto bank-foto"/>
 
                     <div className="about-layout__info">
-                        <p>Żeby zagwarantować naszym użytkownikom pewne i oficjalne kursy wymiany używamy przeliczników
-                            podanych przez <strong>Narodowy Bank Polski</strong> który w każdy dzień roboczy aktualizuje
-                            swoje najpopularniejsze
-                            waluty. Te bardziej egzotyczne aktualizowane są co środę.</p>
+                        <p>{t('AboutSecondPhotoFirstParagraph')}</p>
                     </div>
                 </div>
 
                 <div className="about-layout__item">
                     <div className="about-layout__info">
-                        <p>Nasz konwerter jest przystosowany do działania na każdym urządzeniu. Potrzebujesz coś szybko
-                            przeliczyć? Aplikacja działa wyśmienicie na telefonach komórkowych czy tabletach.</p>
-
-                        <p><strong>CConverter</strong> jest na bieżąco rozbudowywany oraz rozwijany.
-                            Zachęcamy do dzielenia się uwagami przez nasz formularz zgłoszeniowy.</p>
+                        <p>{t('AboutThirdPhotoFirstParagraph')}</p>
```

Comment:
As above, try to use 'Trans' component and wrap up both paragraphs translations into single one in translation file.

Reply:
Done

Reply:
I am not sure what you mean by that. Please explain

Reply:
feel free to merge it bro. will try to refactor it in coming week.